### PR TITLE
Show all cohorts by default

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: viewer/core/static/js/app/components/containers/chart-container.js [hunk 1]
```diff
@@ -228,6 +228,8 @@ class ChartContainer extends React.Component {
     } else {
       return (
         <Chart
+          {...this.props}
```
Comment (spasovski) [inline viewer/core/static/js/app/components/containers/chart-container.js]: Why did you move this? Are we overriding something existing?

**Comment by spasovski** [comment]
Looks good at a glance r+wc. It will create some merge issues for me but shouldn't be too rough.

File: viewer/core/static/js/app/components/containers/chart-container.js (hunk 1 above)
Comment (openjck) [inline viewer/core/static/js/app/components/containers/chart-container.js]: Nothing that I'm aware of. But I've come to feel it's a best practice. I ran into a couple of bugs during development where something from `{...this.props}` was overriding something I manually wrote and it really tripped me up.

When `{...this.props}` comes first, everything manually defined takes precedence.

File: viewer/core/static/js/app/components/views/dataset-control-cohorts.js [hunk 2]
```diff
@@ -1,8 +1,3 @@
-import React from 'react';
-
-
 export default function(props) {
-  return (
-
-  );
+  return null;
```
Comment (spasovski) [inline viewer/core/static/js/app/components/views/dataset-control-cohorts.js]: This file got committed by accident. Feel free to nuke good sir.
